### DISCUSSION TITLE
About sphere model simulation

### DISCUSSION THREAD

**Discussion by lingoodluck** [simnibs/simnibs#478] [9. Results and analysis]
I used the sphere.msh file from the example folder to conduct two simulations. The radius of the sphere model is 95, and the electrode pairs I set are (0, 95, 0), (0, -95, 0); (95, 0, 0), (-95, 0, 0). The current used in both simulations is the same, but the results of the two simulations are not completely identical, the maximum field strengths produced are unequal, which confuses me.Has anyone encountered this situation? Additionally, I tried to generate a sphere model myself, but the coordinate origin defaults to the top left corner, while the coordinate origin of the sphere model in the example is at the center of the sphere. How is this set?
![图片23](https://github.com/user-attachments/assets/6e64e597-9dcc-4921-ae40-59a08e12ba6e)

**Comment by TorgeW** [comment]
Due to the fairly coarse sphere model and therefore also coarse electrode model there will be some numerical error that explains the difference here. The error can be reduced by increasing the element count (decreasing the element size).

If you created the sphere wire generating the mesh from a nifti with SimNIBS then the center is defined via the affine transformation attached to the nifti file.
